include/zephyr/drivers/adc/adc_voltage_divider.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023 The ChromiumOS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_ADC_VOLTAGE_DIVIDER_H_
+#define ZEPHYR_INCLUDE_DRIVERS_ADC_VOLTAGE_DIVIDER_H_
+
+#include <zephyr/drivers/adc.h>
+
+struct voltage_divider_dt_spec {
```

Comment:
This isn't really a driver, so I wonder if this all belongs to `drivers/`...

Reply:
should we start an `include/zephyr/adc` directory for non driver stuff?